apps/web-client/src/routes/orders/+layout.svelte

```diff
@@ -0,0 +1,219 @@
+<script lang="ts">
+	import {
+		BookCopy,
+		Library,
+		PackageMinus,
+		Search,
+		Settings,
+		PersonStanding,
+		AlignLeft as Menu,
+		ArrowLeftToLine,
+		QrCode,
+		Book,
+		Truck
+	} from "lucide-svelte";
+	import { fade, fly } from "svelte/transition";
+	import { createDialog, melt } from "@melt-ui/svelte";
+
+	import { testId, type WebClientView } from "@librocco/shared";
+
+	import { LL } from "$i18n/i18n-svelte";
+
+	import { goto } from "$lib/utils/navigation";
+	import { TooltipWrapper } from "$lib/components";
+	import { appPath } from "$lib/paths";
+	import { getDB } from "$lib/db";
+
+	import { page } from "$app/stores";
+
+	interface Link {
+		label: string;
+		href: string;
+		icon: any;
+		// This is used purely for testing purposes
+		linkto?: WebClientView;
+	}
+
+	$: ({ nav: tNav } = $LL);
+
+	let links: Link[];
+	$: links = [
+		{
+			label: tNav.search(),
+			href: appPath("stock"),
+			icon: Search,
+			linkto: "stock"
+		},
+		{
+			label: tNav.inventory(),
+			href: appPath("inventory"),
+			icon: Library,
+			linkto: "inventory"
+		},
+		{
+			label: tNav.outbound(),
+			href: appPath("outbound"),
+			icon: PackageMinus,
+			linkto: "outbound"
+		},
+		{
+			label: tNav.settings(),
+			href: appPath("settings"),
+			icon: Settings,
+			linkto: "settings"
+		},
+		{
+			label: tNav.history(),
+			href: appPath("history/date"),
+			icon: Book,
+			linkto: "history/date"
+		},
+		{
+			label: "Customers",
+			href: appPath("customers"),
+			icon: PersonStanding,
+			linkto: "orders/customers"
+		},
+		{
+			label: tNav.supplier_orders(),
+			href: appPath("supplier_orders"),
+			icon: Truck,
+			linkto: "orders/suppliers"
+		}
+	];
+
+	export let view: WebClientView;
+	export let loaded: boolean;
+
+	const {
+		elements: { trigger, overlay, content, close: closeMobileMenu, portalled },
+		states: { open: mobilMenuOpen }
+	} = createDialog({
+		forceVisible: true
+	});
+
+	/**
+	 * Handle create note is an `on:click` handler used to create a new outbound note
+	 * _(and navigate to the newly created note page)_.
+	 */
+	const handleCreateNote = async () => {
+		const note = await getDB().db?.warehouse().note().create();
+		await goto(appPath("outbound", note.id));
+	};
+</script>
+
+<div data-view={view} data-loaded={loaded} class="flex h-screen w-screen overflow-hidden">
+	<!-- Sidenav -->
+	<div class="flex">
+		<div class="hidden h-screen bg-gray-800 sm:inline-block">
```

Comment:
CSS like `bg-gray-` and `text-gray-` are messing with the daisy-ui theme. Remove them